src/components/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import { getCommentsById } from "../api";
+import CommentCard from "./CommentCard";
+
+const Comments = ({ article_id }) => {
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState(null);
+
+
+useEffect(() => {
+    getCommentsById(article_id)
+    .then((comments) => {
+        setComments(comments)
+        setIsLoading(false)
+    }).catch((err) => {
+        setError(err.message)
+        setLoading(false)
+    })
+}, [article_id])
```

Comment:
do we need to have article_id in the dependency array? what happens if we run this without it?